checkpoint/schema.json

```diff
@@ -0,0 +1,103 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://spacemesh.io/checkpoint.schema.json.1.0",
+  "title": "checkpoint data",
+  "description": "checkpoint data for recovery",
+  "type": "object",
+  "required": [
+    "version",
+    "data"
+  ],
```

Comment:
do you know if the encoding is more compact?

Reply:
if encoded in binary protobuf, yes it will be more compact. i think all bytes are encoded in hex so thats a double. if base58/base64 thats less than double but still 1.7 maybe

but it probably not worth to change now

Reply:
filed https://github.com/spacemeshos/go-spacemesh/issues/4377